domain/consensus/utils/consensushashing/calculate_signature_hash.go

```diff
@@ -102,6 +103,9 @@ func calculateSignatureHash(tx *externalapi.DomainTransaction, inputIndex int, t
 	sequencesHash := getSequencesHash(tx, hashType, reusedValues)
 	infallibleWriteElement(hashWriter, sequencesHash)
 
+	sigOpCountsHash := getSigOpCountsHash(tx, hashType, reusedValues)
+	infallibleWriteElement(hashWriter, sigOpCountsHash)
```

Comment:
Shouldn't you always sign on your own sigopcount as well?

Reply:
Whooopppsss... good call.